com.toxicstoxm.ledsuite.yml

```diff
@@ -0,0 +1,63 @@
+id: com.toxicstoxm.ledsuite
+branch: '0.1.3'
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
```

Comment:
why do you need the GNOME platform?

Reply:
Because we need the native libadwaita shared library.